cloudify_awssdk/ec2/resources/subnet.py

```diff
@@ -130,6 +140,14 @@ def create(ctx, iface, resource_config, **_):
     iface.update_resource_id(subnet_id)
     utils.update_resource_id(ctx.instance, subnet_id)
 
+    modify_subnet_attribute_args = \
+        _.get('modify_subnet_attribute_args')
```

Comment:
@EarthmanT How the user of the blueprint should be able to know  what to passed in order to modify subnet ? like do you think it is a better add this as an blueprint example ?

Reply:
Same here why we should need to update subnet on the create operation ?

Reply:
This is the pattern that was established with `modify_instance_attribute` and `modify_network_interface_attribute`. I can add to the blueprint example. Some attributes need to be set during install workflow. And rather than dedicate an entire interface operation to modify attribute, we wrap it up in create (since this is the earliest time that it can be applied.)

We will also expose an operation for calling outside of the install workflow. (see `modify_subnet_attribute` operation in [plugin.yaml](https://github.com/cloudify-incubator/cloudify-awssdk-plugin/pull/162/files#diff-3a401e9248f1f2ae5d74a5ce5b735ae9R1639). - Not applied yet. )